wordpress-mobile/WordPress-iOS
# Plans post purchase flow

Implements a first run at the plans post-purchase flow. Includes a simple 'faked' purchase / checkout process (alerts that mimic StoreKit). 

**To Test**

* View the settings a WordPress.com site
* Tap Plans
* Choose a plan
* Tap Purchase
* Confirm the alerts

Be sure to check Premium and Business plan purchases (business includes an extra screen), and check the calls to action within each screen.

**Known Issues**

* It's possible to slightly screw up the scrollview offset when Voice Over is enabled. Select the page control, swipe up / down to the next / previous page, and then very quickly swipe left or right to navigate to the screen content. I spent a while on it but couldn't find a way to adjust this. Navigation still seems to work okay though and it does correct itself, so I figured it wasn't a huge concern at the moment.
* The Plans comparison screens don't update themselves when a plan is 'purchased' – this will happen later when we implement real purchasing.
* This paged scrollview has a lot in common with the plans comparison paged scrollview. I may consolidate some of their features in another PR.
* Vertical positioning of elements may not quite match the design mockups at this stage.
* A larger version of each plan's active image is required.

See #4746
Needs Review: @koke
